node/src/components/linear_chain_sync/state.rs

```diff
@@ -39,14 +41,14 @@ pub enum State {
         validators_for_latest_block: BTreeMap<PublicKey, U512>,
     },
     /// Synchronizing done.
-    Done,
+    Done(Option<Box<Block>>),
```

Comment:
If we're leaving this `Option<Box<Block>>`, please add a comment to the enum variant what it stands for.